pkg/rancher-desktop/generated/README.md

```diff
@@ -1,3 +0,0 @@
-# Generated Files
```

Comment:
This seems to be unrelated to the rest of the PR, but I guess is fine because it doesn't deserve its own PR.

Reply:
It's because the generated file actually tripped the spell checker locally (‽)